src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  max-width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-between;
+  padding-left: 10px;
+
+  &__day {
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+    width: $day-size;
+    height: $day-size;
+    line-height: $day-size - 2px;
```

Comment:
There is an issue with the line-height calculation. The expression `$day-size - 2px` should be enclosed in parentheses to ensure correct calculation. It should be `line-height: ($day-size - 2px);`.